package.json

```diff
@@ -29,14 +29,14 @@
     "toys": "1.x.x"
   },
   "peerDependencies": {
-    "hapi": ">=10 <17",
+    "hapi": ">=10 <=17",
```

Comment:
We will likely follow hapi and drop support for hapi v10 thru v16.

Reply:
ah, gotcha, good to know. I'll update that, then